plugins/catalog/src/components/CatalogFilter/CatalogFilter.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {
+  Card,
+  List,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+
+export type CatalogFeatureItem = {
+  id: string;
+  label: string;
+  icon?: React.ReactElement;
+  count?: number;
+};
+
+export type CatalogFilterGroup = {
+  name: string;
+  items: CatalogFeatureItem[];
+};
+
+export type CatalogFilterProps = {
+  groups: CatalogFilterGroup[];
+};
+
+const useStyles = makeStyles({
+  root: {
+    backgroundColor: 'rgba(0, 0, 0, .11)',
+    boxShadow: 'none',
+  },
+  title: {
+    margin: '10px 0 0 10px',
```

Comment:
We try to consistently use the theme spacing everywhere: `theme.spacing(1, 0, 0, 1)` here I guess